src/autorest-core/lib/configuration.ts

```diff
@@ -37,6 +37,8 @@ export interface AutoRestConfigurationImpl {
   "fluent"?: boolean;
   "azure-arm"?: boolean;
```

Comment:
switching between what was formerly known as "CSharp" and "CSharp.Azure" or "Java" and "Java.Azure" is done with that flag